server/src/RelayHttpServer.go

```diff
@@ -185,6 +187,7 @@ func parseCommandLine() (relayParams librelay.RelayParams) {
 	ethereumNodeUrl := flag.String("EthereumNodeUrl", "http://localhost:8545", "The relay's ethereum node")
 	workdir := flag.String("Workdir", filepath.Join(os.Getenv("PWD"), "build/server"), "The relay server's workdir")
 	flag.BoolVar(&shortSleep, "ShortSleep", false, "Whether we wait after calls to blockchain or return (almost) immediately")
+	flag.BoolVar(&devMode, "DevMode", false, "Enable developer mode (do not retry unconfirmed txs, do not cache account nonce)")
```

Comment:
I think it can be merged with ShortSleep, which is our initial code for test-enabled relay..

Reply:
Sounds good! I was tempted to do it, but didn't want to introduce that breaking change. Should I remove ShortSleep then in favor of DevMode?

Reply:
Yes